### ISSUE TITLE
[BUG] NoRotate Desync

### ISSUE THREAD

**Issue by linustouchtips** [bebeli555/CookieClient#13]
**Issue:**

The issue is that the NoRotate module will desync your client and cause you to get kicked when in a burrow on crystalpvp.cc; this is because of their retarded packet fly patch that basically sends a million SPacketPlayerPosLook packets every second that you are inside a block. 

The module should not be canceling the event because that causes the server to think that you are packet flying and kick you. 

**Recreate:** 

1. Join crystalpvp.cc
2. Turn on burrow
3. Wait a few seconds in the burrow
4. Server disconnects you

**Comment by bebeli555** [comment]
Ill try to fix it next version, thanks.

**Comment by linustouchtips** [comment]
sounds good, thanks

**Comment by bebeli555** [comment]
Should be fixed now

**Comment by linustouchtips** [comment]
swag
